README.md

```diff
@@ -31,9 +31,37 @@ Requires `node` version 8.3.0 and `npm` 5.3.0. Consider using [Node Version Mana
 Run `npm install`. 
 This will download all the required dependencies to be able to start the UI.
 
-
 Run `npm start`. This will start the UI with live reload enabled. Then navigate to <http://localhost:3000>.
 
+### Run test
+
+We use [jest](https://github.com/facebook/jest) test loader because it's faster than karma execution.
+
+#### All tests
+```
+npm run test
```

Comment:
`test` is a keyword in npm and therefore we don't need the `run` command. `npm test` works and is shorter.

Reply:
Hah. I always still type `npm run test`